README.md

```diff
@@ -1,2 +1,232 @@
-# module-ballerinax-aws.redshiftdata
-AWS Redshift Data API connector for Ballerina.
+# AWS Redshift Data API connector for Ballerina.
+
+[![Build](https://github.com/ballerina-platform/module-ballerinax-aws.redshiftdata/actions/workflows/ci.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-aws.redshiftdata/actions/workflows/ci.yml)
+[![Trivy](https://github.com/ballerina-platform/module-ballerinax-aws.redshiftdata/actions/workflows/trivy-scan.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-aws.redshiftdata/actions/workflows/trivy-scan.yml)
+[![codecov](https://codecov.io/gh/ballerina-platform/module-ballerinax-aws.redshiftdata/branch/main/graph/badge.svg)](https://codecov.io/gh/ballerina-platform/module-ballerinax-aws.redshiftdata)
+[![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerinax-aws.redshiftdata.svg)](https://github.com/ballerina-platform/module-ballerinax-aws.redshiftdata/commits/main)
+[![GraalVM Check](https://github.com/ballerina-platform/module-ballerinax-aws.redshiftdata/actions/workflows/build-with-bal-test-graalvm.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-aws.redshiftdata/actions/workflows/build-with-bal-test-graalvm.yml)
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+
+[Amazon Redshift](https://docs.aws.amazon.com/redshift/latest/mgmt/welcome.html) is a powerful and fully-managed data warehouse service provided by Amazon Web Services (AWS), designed to efficiently analyze large datasets with high performance and scalability.
+
+The `ballerinax/aws.redshiftdata` package allows developers to interact with Amazon Redshift Data API seamlessly using Ballerina. The Redshift Data API simplifies data access by eliminating the need for managing persistent database connections or the Redshift JDBC driver.
+
+### Usage Options
+
+The Redshift Data Ballerina Connector can be used in two ways:
+
+1. **Redshift Cluster**: Connect directly to a provisioned Amazon Redshift cluster.
+2. **Serverless Mode**: Leverage the serverless capabilities of the Redshift Data API, which is particularly suited for applications that need to scale dynamically without managing infrastructure.
```

Comment:
Should not this be included as a subsection in the setup guide on how to retrieve connection configs for cluster or a serverless workgroup ?